webapi/src/main/scala/org/knora/webapi/util/InputValidation.scala

```diff
@@ -333,10 +334,6 @@ object InputValidation {
       * @return the same string.
       */
     def toDate(s: String, errorFun: () => Nothing): String = {
-        // TODO: how to deal with dates BC -> ERA
-
-        // TODO: import calendars instead of hardcoding them
```

Comment:
> TODO: import calendars instead of hardcoding them

Probably I wrote this TODO. I do not remember what it was supposed to mean. Maybe that we hardcoded JULIAN and GREGORIAN calendars? How would include an additional calendar (could we make this configurable)?